Cargo.toml

```diff
@@ -22,3 +22,4 @@ edition = "2021"
 
 [dependencies]
 chrono = { version = "0.4.35", default-features = false }
+defmt = { version = "0.3", optional = true }
```

Comment:
```suggestion
defmt-03 = { version = "0.3", optional = true }
```

Reply:
Why this? Genuine question, I have seen crates doing this and some not.

Reply:
That is so that we can add support for another version of defmt without a breaking change.
That was not the right way to do it, though. I have reverted the change.
Could you add a feature `defmt-03` which enables the `defmt` dependency?